jest.config.js

```diff
@@ -22,4 +22,5 @@ module.exports = {
   coveragePathIgnorePatterns: ['node_modules'],
   setupFilesAfterEnv: ['jest-extended/all', '../jest.setup.js'],
   globalSetup: '../jest.global-setup.js',
+  prettierPath: null,
```

Comment:
Jest 29 is incompatible with Prettier v3, see https://github.com/jestjs/jest/issues/14305.